test/integration/scheduler/scheduler_test.go

```diff
@@ -162,6 +162,70 @@ func TestSchedulerCreationFromConfigMap(t *testing.T) {
 			),
 			expectedPrioritizers: sets.NewString(),
 		},
+		{
```

Comment:
This actually doesn't cover the changes this PR introduced - it's creating a ConfigMap through apiserver.

What the test needs to cover is: https://github.com/kubernetes/kubernetes/blob/a9db137737566d2ebe4e578c80a3745e3cc01aa8/pkg/scheduler/scheduler.go#L203-L204

Writing a simple unit test should be good enough.

Reply:
These test cases used for yaml format config in ConfigMap which is not supported before this PR due to the codec is same as `initPolicyFromFile`.

Thanks for your tips, I added a unit test for `initPolicyFromFile`. PTAL